### ISSUE TITLE
Request to add back option to pass updateMathImg as an input

### ISSUE THREAD

**Issue by RGunning** [digabi/rich-text-editor#620]
Previously with the old rich-text-editor it was possible to configure the updateMathImg passes to makeRichText and use that to convert the latex formula to a base64 image without sending the request to a nodeJS server. 

```ts
 makeRichText(
    editorElement,
    {
        baseUrl: '',
        updateMathImg: ($img, latex) => {
            $($img).prop({
                src: 'data:image/svg+xml;base64,' + this.getSvgFromLatex(latex),
                alt: latex.replace(/</g, '\\lt ').replace(/>/g, '\\gt '),
            });
            $($img).closest('[data-js="answer"]').trigger('input');
        },
    },
    this.emitChange
);

function getSvgFromLatex(latex: string) {
        try {
            const node = this.convertLatexToSvg(latex);
            if (node.includes('data-mml-node="merror"')) {
                return this.getLatexErrorSvg();
            }
            return window.btoa(
                encodeURIComponent(node).replace(/%([0-9A-F]{2})/g, (_match, p1) =>
                    String.fromCharCode(Number('0x' + p1))
                )
            );
        } catch {
            return this.getLatexErrorSvg();
        }
    }

const mathjaxDocument = mathjax.document('', {
    InputJax: new TeX({ packages: AllPackages }),
    OutputJax: new SVG({ fontCache: 'local' }),
});

function convertLatexToSvg(latex: string): string {
        const mathjaxOptions = {
            em: 16,
            ex: 8,
            containerWidth: 1280,
        };
        const filteredLatex = latex.replace(/€/g, '\\texteuro');

        return adaptor.innerHTML(mathjaxDocument.convert(filteredLatex, mathjaxOptions));
    }
```

With the new rich-text-editor, that option seems to be gone, so it is only possible to set the baseUrl of the server that will process the latex formula. Please re-add an option so we can process the latex formula in the browser and not need to run a server to process latex formula conversion requests.

**Comment by jan-aleksi** [comment]
You can now pass a `onLatexUpdate` callback to `makeRichText` that overrides default latex update handler. It lets you set whatever attributes you need on the image element. I hope it solves your use case!

This is what the default handler does:
```ts
(img: HTMLImageElement, latex: string) => {
  img.setAttribute('src', `${baseUrl}/math.svg?latex=${encodeURIComponent(latex)}`)
  img.setAttribute('alt', latex)
}
```
So even if you only want to replace the `src` attribute, your custom `onLatexUpdate` callback needs to set the `alt` attribute as well.

**Comment by Dunos** [comment]
@jan-aleksi thank you, this is working as expected, but there is an issue when we need to initialize the the editor with content previously saved (using images with a modified src). The **initMathImages** method is looking only for images with _/math.svg_ so it gets a bit messy to replace and init the images

**Comment by jan-aleksi** [comment]
Right, sorry about that. If `initMathImages` would work for elements with `img[src^="data:image/svg+xml"]` selector, would that work for you? I'm going to add a more robust way to detect math images, but that won't work for existing data.

**Comment by Dunos** [comment]
@jan-aleksi probably yes, not sure if that would break something for the normal use case that I see it is checking the current origin of the image with the `baseUrl` param. We just need the existing images (meaning the ones we pass to the `initialValue`) to be initialized so when they are clicked, it opens the formula editor, and that is happening in the `createMathImage` call inside the `initMathImages `

**Comment by jan-aleksi** [comment]
@Dunos what I tried to ask was if all of your existing images have their src-attribute start with `data:image/svg+xml` so that I can fix the initialization for all of your existing data.

As a proper fix for using custom src-attribute values I'm adding a new `data-math-image` attribute that is used to select the images that should be initialized as editable math images, but that obviously won't work for existing data.

**Comment by Dunos** [comment]
@jan-aleksi sorry my mistake. Yes, all the images have that src attribute starting with `data:image/svg+xml;base64,`

**Comment by jan-aleksi** [comment]
@Dunos version 8.6.0 has now been released, which fixes the math image initialization. I hope everything works as expected now.

**Comment by Dunos** [comment]
@jan-aleksi seems to be working fine now, but we'll test a bit more deeply the next days, thanks!

**Comment by jan-aleksi** [comment]
Alright, I'll close this issue then, but feel free to reopen it, if you have more issues with this change.
